independent-publisher-connectors/Mockaroo/apiDefinition.swagger.json

```diff
@@ -0,0 +1,114 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Mockaroo",
+    "description": "Mockaroo can be used to generate realistic sample data to help test applications",
+    "contact": {
+      "name": "Richard Wilson",
+      "email": "richard.a.wilson@microsoft.com",
+      "url": "https://www.richardawilson.com/"
+    }
+  },
+  "host": "api.mockaroo.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/api/generate.json": {
+      "post": {
+        "summary": "Generate data from existing schema",
+        "description": "Returns generated data from an existing schema defined in Mockaroo.",
+        "operationId": "GenerateDataFromExistingSchema",
+        "parameters": [
+          {
+            "$ref": "#/parameters/array"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "schema": {
+                  "type": "string",
+                  "x-ms-summary": "Schema Name",
+                  "description": "Enter name of the Mockaroo schema",
```

Comment:
@rwilson504 please make sure that descriptions are grammatically correct and are free of typographical errors. More information can be found in [English strings guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).